internal/config/config_util.go

```diff
@@ -33,6 +35,19 @@ func OverrideWithLoggingFlags(mountConfig *MountConfig, logFile string, logForma
 	}
 }
 
+type cliContext interface {
```

Comment:
Good to write one liner, this is abstraction over the IsSet() method of cli.Context. Specially added to keep the unit test simple.